task/pluginversioncheck.go

```diff
@@ -39,13 +40,21 @@ func showPluginUpdateNotification(reports []plugin.VersionCheckReport) {
 	}
 	longestNameLength := 0
 	for _, report := range reports {
-		thisName := fmt.Sprintf("%s/%s", report.CheckResponse.Org, report.CheckResponse.Name)
+		thisName := getNameFromReport(report)
```

Comment:
why not add a Name function to the report struct 
then this could just be 
```
if nameLength := len(report.Name); nameLength > longestNameLength {
			longestNameLength = nameLength
		}
```